src/crypto/api.ts

```diff
@@ -19,7 +19,9 @@ import { IKeyBackupInfo } from "./keybackup";
 import { GeneratedSecretStorageKey } from "../crypto-api";
 
 /* re-exports for backwards compatibility. */
-export { CrossSigningKey, GeneratedSecretStorageKey as IRecoveryKey } from "../crypto-api";
+// CrossSigningKey is used as a value in `client.ts`, we can't export it as a type
```

Comment:
It could also be used as a value by downstream consumers, not sure this comment adds value

Reply:
Happy either way, the lint rule will help out also